src/modules/servers/infrastructure/repositories/__tests__/server.typeorm.repository.spec.ts

```
@@ -0,0 +1,64 @@
+import { Test } from '@nestjs/testing';
+import { DataSource } from 'typeorm';
+import { ServerTypeormRepository } from '../server.typeorm.repository';
+import { Server } from '../../../domain/entities/server.entity';
```

ESLint

[error] 'Server' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)